### ISSUE TITLE
Party Members Overlay has names write over chat

### ISSUE THREAD

**Issue by san7890** [Wynntils/Wynntils#2160]
Not sure if this is just me or not, but posting it here just in case.

On v0.0.4-beta.8

Here are some photographs with the behavior in question:
This is what the party member overlay looks like:
![image](https://github.com/Wynntils/Artemis/assets/34697715/8877e92a-ba1d-4b9f-88f1-e54816e60ca9)

And this is what it looks like when I open up the chat:
![image](https://github.com/Wynntils/Artemis/assets/34697715/02066e4a-d5d2-4190-bcf9-29f4f0cd30b9)

The names are writing over chat for some reason. I do not believe this happened in any versions prior to v0.0.4. I personally would expect behavior to have the whole overlay be temporarily disabled rather than have some artifacts look weird in the chat window (or at least have the chat window write over everything). There's probably a better fix than what I'm thinking of but would be nice if this clash could be resolved.

**Comment by magicus** [comment]
You can move the party overlay so it does not collide with the chat window.

**Comment by magicus** [comment]
It's a bit odd that just the name is showing, and not the faces and bars, though...

**Comment by san7890** [comment]
Yeah, this is completely new behavior. Would rather whatever senses to hide the faces+bars would also hide the names (text)

**Comment by ShadowCat117** [comment]
There's also some issues with the player list overlay I think, party mana bar renders above it but not the rest

**Comment by ShadowCat117** [comment]
Does not appear to occur anymore

![Image](https://github.com/user-attachments/assets/bfbdb845-f17e-487a-9baa-4729bec9f1e2)

Though there is the larger issue of we should be able to determine the "layer" of overlays, so users can choose to have overlays render over other ui elements/overlays
